Scripts/Python/xJournalBookGUIPopup.py

```diff
@@ -128,12 +140,21 @@ def OnNotify(self, state, id, events):
                         PtDebugPrint("xJournalBookGUIPopup:Book: NotifyShow",level=kDebugDumpLevel)
                         # disable the KI
                         PtSendKIMessage(kDisableKIandBB,0)
+                        if CurrentPage > -1:
+                            JournalBook.open(CurrentPage) #Opens to saved page
+                            JournalBook.goToPage(CurrentPage) #shows page tabs at bottom when used after open
```

Comment:
Looks like we may need to fix a bug in `pfJournalBook`. Is it that the page turn buttons are missing?

Reply:
Yes, during my testing both commands were needed to consistently open the book to the correct page and show the tabs.